mods/coop/hook/lua/sim/ScenarioUtilities.lua

```diff
@@ -1,12 +1,18 @@
 -- Ignore unit restrictions in all functions that are spawning units
 
 local ScenarioFramework = import('/lua/ScenarioFramework.lua')
+local isCommonArmy = nil
```

Comment:
initiazing variable with nil value has no point, why don't you check if the settings is enabled here and then you don't need to check in every function

Reply:
ScenarioInfo is not yet available here.